analysis.py

```diff
@@ -213,6 +217,52 @@ def gpgsaParse(self, gpgsa, line):
         if self.verifyChecksum(line, gpgsa[7]):
             return gpgsa
 
+        '''
+
+        gpgsv[0] - "GPS Satellites in View"
+        gpgsv[1] - Number of sentences for full data
+        gpgsv[2] - The sentence number
+        gpgsv[3] - The number of satellites in view
+        gpgsv[4] - Information on satelite in view
+                   [Information on Satellite PRN number; Elevation in  degrees, 90 maximum; Azimuth, degrees from true north, 000 to 359; SNR 00-99db (null when not tracking)]
+        gpgsv[5] - Information about second SV, same as gpgsv[4]
+        gpgsv[6] - Information about second SV, same as gpgsv[4]
+        gpgsv[7] - Information about second SV, same as gpgsv[4]
+        gpgsv[8] - Checksum
+        *Depending on the amount of satellites in view, gpgsv[5-7] may not exist. In that case, the checksum will equal the length(gpgsv)-1 
```

Comment:
well, it doesn't equal the length, but the location of the checksum in the array is at ```gpgsv[len(gpgsv)-1]```